doc/plugins/org.polarsys.capella.preferences.doc/html/06. Preferences/6.2. Model preferences.html

```diff
@@ -95,24 +99,71 @@ <h3 id="Title_Block">Title Block</h3>
 			<li>When a cell is selected, the content of the cell is displayed on the bottom of the table.</li>
 		</ul>
 		<p>
-			<img border="0" src="Images/Title_Block_DiagramPreferences.PNG"/>
+			<img height="595" width="877" border="0" src="Images/Title_Block_DiagramPreferences.PNG"/>
+		</p>
+		<ul>
+			<li>The content of each cell can be editable by double clicking on a cell.</li>
+		</ul>
+		<p>
+			<img height="595" width="877" border="0" src="Images/Title_Block_CellDefault_DiagramPreferences.PNG"/>
+		</p>
+		<p>
+			<br/>
+			A new dialog is opened the fields below can be edited (we can edit the existing fields or add new values for the new cells):
 		</p>
 		<ul>
-			<li>The content of each cell can be editable by double clicking on a cell. A new dialog is opened and can edit the fields below:
+			<li>name: enter the name(label) that will be displayed on the cell</li>
```

Comment:
**name(label)** -> **name (label)**

Reply:
... will be displayed as the **header** of the cell

Reply:
Done